ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/pre-external-lb/provision/cucushift-installer-rehearse-vsphere-ipi-pre-external-lb-provision-chain.metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+	"path": "cucushift/installer/rehearse/vsphere/ipi/pre-external-lb/provision/cucushift-installer-rehearse-vsphere-ipi-pre-external-lb-provision-chain.yaml",
```

Comment:
Can we re-organize the `vsphere/ipi/external-lb` folder to create "pre" and "post" sub folders to maintain the two cases.

Reply:
i created the "pre" and "post" dir under  vsphere/ipi/external-lb folder and updated the affected job accordingly, please help review.